tools/nudgeunix.c

```diff
@@ -140,7 +140,7 @@ main(int argc, const char *argv[])
         return usage();
 
     /* construct the payload */
-    success = create_nudge_signal_payload(&info, action_mask, client_id, client_arg);
+    success = create_nudge_signal_payload(&info, action_mask, 0, client_id, client_arg);
```

Comment:
Is zero a proper value in the flags enum?

Reply:
See earlier reply: this is not an enum typed parameter but an integer.